front-end/src/screens/Post/Post.tsx

```diff
@@ -157,12 +162,12 @@ export default styled(Post)`
 		}
 
 		code {
-			padding: 0.2em 0 0.2em 0;
+			padding: 0.2rem 0 0.2rem 0;
 			margin: 0;
-			font-size: 1.2rem;
+			font-size: sm;
 			background-color: rgba(0, 0, 0, 0.04);
 			border-radius: 3px;
-			color: #555252;
+			color: text;
```

Comment:
Missed this in a previous review I guess, `text` should rather be `black_text` or something that at least has a color in the name.

Reply:
good call - updating now